### ISSUE TITLE
LoRA fine-tuning of DINOv3 on more tasks

### ISSUE THREAD

**Issue by Fly-Yun-Maker** [RobvanGastel/dinov3-finetune#8]
Thank you for your outstanding contributions. May I ask if you have applied LoRA fine-tuning to DINOv3 for classification and depth estimation to achieve better experimental results? Once again, thank you for your excellent work, and I look forward to your response.

**Comment by Fly-Yun-Maker** [comment]
Thank you!

**Comment by RobvanGastel** [comment]
I just started checking the differences in results. What are the settings you're testing the finetuning with? For better performance we should probably also consider a learning rate scheduler with cosine annealing with warm-up. Also the input resolution I use might be different from the paper.

In table 14 of their paper is the only part where I see them compare the ViT-S model, but I am not sure how they finetuned their model there. If you share your setup lets see how we can improve the performance.

**Comment by RobvanGastel** [comment]
@Fly-Yun-Maker  Afer double checking my metrics, I tested different size ViT models with the linear head to verify the performance. I finetuned with LoRA versus them only finetuning the linear decoder and probably a different input image size as they dont mention this size. Their results comes from Table 3, this is the table I obtained: 

<img width="1760" height="1101" alt="Image" src="https://github.com/user-attachments/assets/d6d039e1-0f17-4f1c-a247-fb431b6689c4" />

 Since all the smaller models are distilled these might also behave slightly different. Therefore I think the results are similar enough to the paper. I will push the LoRA weights in the next commit. If you have more questions or comments feel free to re-open the issue and ask.
